workshops/templates/workshops/instructors.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block content %}
-    {% crispy form form_helper %}
+    {% crispy form %}
 {% if persons == None %}
```

Comment:
Can you please explain to me why is this conditional unnecessary? As far as I can understand, it works exclusively for GET requests and renders nothing below the form (no table headers).  
Am I mising something?